app/functions.sh

```diff
@@ -102,6 +124,7 @@ reload_nginx() {
             echo "Reloading nginx proxy (${_nginx_proxy_container})..."
             docker_exec "${_nginx_proxy_container}" \
                         '[ "sh", "-c", "/usr/local/bin/docker-gen /app/nginx.tmpl /etc/nginx/conf.d/default.conf; /usr/sbin/nginx -s reload" ]'
+            [[ $? -eq 1 ]] && echo "$(date "+%Y/%m/%d %T"), Error: can't reload nginx-proxy. Check if the container is running" >&2
```

Comment:
I think the "Check if the container is running" on this error message is redundant, as we'll get it from the `docker_exec` error just before that, right ?

Reply:
Yes, you are right.